### ISSUE TITLE
Bug / 

### ISSUE THREAD

**Issue by mgolpashin** [porscheui/porsche-design-system-contribution#40]
**Description**  

When using a PIcon inside a ScrollContainer of react-sortable-hoc, the Icon falls back to the default Icon while dragging.

**Expected behavior**  
The correct Icon is still displayed while dragging

**Screenshots**  
![imageManager](https://user-images.githubusercontent.com/3465250/83752628-21f98900-a669-11ea-9c5a-e1be21af5e1a.gif)

I am using react and react-dom@16.13.0 with react-sortable-hoc@1.11.0.

**Comment by denyo** [comment]
Small hint: The element being dragged is not the element where the dragging starts. It's a duplicate or new element that is added to the DOM while dragging. The old element becomes invisible during this time.

**Comment by mgolpashin** [comment]
@denyo thats true. I was able to debug the behavior. Still, I don't see why the Icon of the dragged element is using the fallback svg path. Maybe it was something to do with the icons being lazy loaded? I remember there was another issue with that in v1.3.0

![Bildschirmfoto 2020-06-05 um 14 59 36](https://user-images.githubusercontent.com/3465250/83879418-dfee4700-a73d-11ea-8cb8-0e15957c13ae.png)

**Comment by o-hook** [comment]
Hi @mgolpashin could you please update to our latest v1.4.x and check if this issue still exists? If so, you can alos try our latest RC release v1.5.0.rc-0 . Thanks!

**Comment by mgolpashin** [comment]
@o-hook Unfortunately I was already using v1.4.0. The upgrade to v1.5.0-rc-0 did not solve the issue nor did the change from PIcon to PButtonBure.
